### PR TITLE
feat: Possibility of execute the interpreter for read a file and get its tokens

### PR DESCRIPTION
**Pull request by FRostri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 # FakeOS-lang
 Un lenguaje de programación amigable cuyo propósito es enseñar a los usuarios nuevos en el mundo de la informática lo básico de los lenguajes de programación y el uso que se les puede dar, es bástante legible y rápido.
+
+You can executate the interpreter for now like this: `./ezcript`
```
Comment (Sebastian-byte) [inline README.md:4]: execute*

File: ezcript [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        print("Welcome to Ezcript v0.0.1!")
+        print("Write \"help()\" for more information")
+
+        repl()
+    else:
+        for arg in args:
+            if not os.path.isfile(
+                    arg) and arg[0:2] != '--' and arg[0:1] != '-':
+                print("El archivo no existe")
+            elif arg[-3:] != '.ez' and arg[0:2] != '--' and arg[0:1] != '-':
```
Comment (Sebastian-byte) [inline ezcript:28]: I think it should support any file extension.

File: ezcript_lang/repl.py [hunk 3]
```diff
@@ -1,21 +1,35 @@
...
 
 
-def start_repl() -> None:
+def clear() -> None:
+
+    # command for windows
+    if name == 'nt':
+        system('cls')
+
+    # commanf for mac and linux
```
Comment (Sebastian-byte) [inline ezcript_lang/repl.py:19]: command*, macOS*

File: ezcript_lang/repl.py [hunk 4]
```diff
@@ -1,21 +1,35 @@
...
     TokenType,
 )
 
 EOF_TOKEN: Token = Token(TokenType.EOF, '')
 
 
-def start_repl() -> None:
+def clear() -> None:
+
+    # command for windows
```
Comment (Sebastian-byte) [inline ezcript_lang/repl.py:15]: Windows* (capitalize it).

**Review by Sebastian-byte — CHANGES_REQUESTED** [review]

File: ezcript (hunk 2 above)
Comment (FRostri) [reply ezcript:28]: You think so? Couldn't we get into trouble later on?

Comment (Sebastian-byte) [reply ezcript:28]: Many compilers and interpreters ignores the extension, like GCC, Clang or Python.

Comment (Sebastian-byte) [reply ezcript:28]: A warning should be ok, not a error.

Comment (FRostri) [reply ezcript:28]: Ooh, all right, I'll leave it then

**Comment by FRostri** [comment]
@Sebastian-byte Is ready

**Review by Sebastian-byte — APPROVED** [review]

**Comment by Sebastian-byte** [comment]
Perdón presione el botón por accidente 😭

**Comment by FRostri** [comment]
ae, no te preocupes ya la vuelvo a crear XD

**Pull request merged**